amida-tech/mcp-data-client
# Feature/mcp 148 jf filenames

### Related tickets:
MCP-148 

Good news @TylerNoblett, your favorite color is back.

### Manual test cases?
Run `yarn test`. Try submitting the new `organs-of-special-sense-vision.xlsx` file alongside the others and check that you get a message for a `no_json_mapping` error. 

### Any additional context/background?
The front-end changes per the backend ticket located [here.] (https://github.com/amida-tech/mcp-data-mapping-utils/pull/68)

### Screenshots (if appropriate):
-
